sdk/harambe/handlers.py

```diff
@@ -78,9 +79,14 @@ class UnnecessaryResourceHandler(AbstractHandler):
     async def handle(self, route: Route) -> None:
         resource_type = route.request.resource_type
         url = route.request.url
-
-        if (
-            resource_type in ["image", "media", "font"]
+        if resource_type in ["image", "media"]:
+            fake_img = "iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII="
```

Comment:
Since this is constant may aswell store the b64 version of this. I would put it at the top as FAKE_1x1_img